cpp/src/slice2cs/CsUtil.cpp

```diff
@@ -282,7 +282,7 @@ Slice::CsGenerator::isValueType(const TypePtr& type)
         DataMemberList dm = s->dataMembers();
         for (const auto& i : dm)
         {
-            if (!isValueType(i->type()) || i->defaultValueType())
+            if (!isValueType(i->type()) || i->defaultValue())
```

Comment:
This does not change anything. I think it's just preferable to consistently check defaultValue() or defaultValueType()... and now all the code checks defaultValue() to see if there is a default value.